Launcher/Launcher/src/Helpers/InstallationHelper.cs

```diff
@@ -0,0 +1,101 @@
+﻿using System.Reflection;
+using WslApiAdapter.WslApi;
+
+namespace Launcher.Helpers;
+
+public static class InstallationHelper {
+    /// <summary>
+    ///     Registers the distribution and runs first-time setup
+    /// </summary>
+    /// <returns>0 on success, an error code otherwise</returns>
+    public static int Install() {
+        Console.WriteLine($"Registering {DistributionInfo.DisplayName}...");
+
+        if (WslApiLoader.WslIsDistributionRegistered(DistributionInfo.Name)) {
+            Console.Error.WriteLine($"{DistributionInfo.DisplayName} is already installed!");
+            return 0;
+        }
+        
+        const string tarFileName = "nixos-wsl-installer.tar.gz";
+
+        // Determine tarball location
+        var assemblyPath = Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location);
```

Comment:
In the Release builds the tar file is included in the Msix package. That means, that it will always be at `../nixos-wsl-installer.tar.gz` relative to `Launcher.dll`. Using `Application.StartupPath` instead of the assembly location might be the better idea for debug builds